process/templates/WG_NAME_incubating_stage.md

```diff
@@ -1,21 +1,36 @@
 ## Working Group incubation application
 
+### Working Group (WG) has met all Sandbox requirement
+  * "link to sandbox PR"
+
 ### List of regular contributors
 The WG must have a minimum of 5 contributors from at least 3 different organizations attending regulary.
-
   * "name, affiliation, GitHub ID"
 
-### TAC sponsor
-The WG must have a TAC sponsor:
-
-  * "name, affiliation, GitHub ID"
 
 ### Mission of the Working Group
 The WG must have a charter or mission statement for review by TAC
-
   * Link to the WG charter or mission statement defining its goals.
 
-### Working Group activity
-The WG must have met at least 5 times
-
+### Governance
+Working Groups must have documented, initial WG governance
+  * "link to governance documents/Charter"
+WG must have met publicly at least 5 times
    * Link to public meeting notes (or ideally recordings)
+WG must have a TAC sponsor
```

Comment:
Same as: https://github.com/ossf/tac/pull/276#discussion_r1509363923

Reply:
+1, I would even suggest an "advisor" at the sandbox stage